protocol-test-util/src/main/java/io/camunda/zeebe/test/broker/protocol/record/ProtocolFactory.java

```diff
@@ -0,0 +1,274 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.test.broker.protocol.record;
```

Comment:
🔧 Consider moving this class up 1 package level to `io.camunda.zeebe.test.broker.protocol`. I think this might make it better clear that this is a class that should be used from outside this module. Introducing the jdk9 module-info would also be a good way to clarify this.